src/modules/stack_overflow.py

```diff
@@ -0,0 +1,80 @@
+import discord.ext.commands as commands 
+import discord.embeds as embed
+import discord.colour as colour
+import requests, stackexchange 
+from logger import ErrorLevel
+from bs4 import BeautifulSoup
+
+''' ISSUES: 
+1) beautiful soup removes all links (<a href>) in answer_body
+2) include question body as well?  
+'''
+class StackOverflow(commands.Cog):
```

Comment:
Does this have to be a class due to discords api?(sorry im a discord api noob)

Reply:
I wrote it using oop because the other modules were. I'm not sure if it's actually necessary

Reply:
yeah thats fair, I just dislike OOP unless its needed

Reply:
thats just needed for the actual command to discord interface though right? the rest of it can be pure functions